package.json

```diff
@@ -107,7 +107,7 @@
     "@storybook/react": "^6.0.21",
     "@storybook/theming": "^6.0.21",
     "@testing-library/jest-dom": "^5.11.6",
-    "@testing-library/react": "^11.2.2",
+    "@testing-library/react": "13.3.0",
```

Comment:
generally speaking would prefer if this was a separate PR for rollback purposes, but i wont block on it.

Reply:
ah, true! I figured it was lower risk since it was a dev dependency/testing package, but I can revert and follow it up. I think the test output is just a bit spammy without the upgrade, but it should still pass

Reply:
not a blocker, just something to consider for future work!